backend/app/views/spree/admin/products/_form.html.erb

```diff
@@ -173,7 +173,7 @@
         </div>
       <% else %>
         <div id="shipping_specs" class="row">
-          <% [:height, :width, :depth, :weight].each_with_index do |field, index| %>
+          <% [:height, :width, :depth, :weight, :gtin].each_with_index do |field, index| %>
```

Comment:
Gtin is an identifier and not a shipping spec, it belongs close to other identifiers such as SKU

Reply:
FYI: 
GTIN = Global Trade Item Number

You know them, for example EAN is a GTIN, they look like this: 
![image](https://github.com/user-attachments/assets/0f67f87a-481b-4ad6-b5d2-999c76ab6b67)